picard/util/tags.py

```diff
@@ -165,653 +346,1176 @@ def names(self, selector=None):
     ),
     TagVar(
         'lyricist',
         shortdesc=N_('Lyricist'),
+        longdesc=N_('The names of the lyricists for the associated work.'),
+        is_multi_value=True,
+    ),
+    TagVar(
+        'lyricistsort',
+        shortdesc=N_('Lyricist Sort'),
+        longdesc=N_('The sort names of the lyricists for the associated work.'),
+        is_multi_value=True,
+        is_hidden=True,
     ),
     TagVar(
         'lyrics',
         shortdesc=N_('Lyrics'),
+        longdesc=N_('The lyrics for the track.'),
+        is_from_mb=False,
     ),
     TagVar(
         'media',
         shortdesc=N_('Media'),
+        longdesc=N_('The media on which the release was distributed (e.g.: CD).'),
     ),
     TagVar(
         'mixer',
         shortdesc=N_('Mixer'),
+        longdesc=N_('The names of the "*Mixed By*" engineers associated with the track.'),
+        is_multi_value=True,
     ),
     TagVar(
         'mood',
         shortdesc=N_('Mood'),
+        longdesc=N_('The mood associated with the track.'),
+        is_from_mb=False,
     ),
     TagVar(
         'movement',
         shortdesc=N_('Movement'),
+        longdesc=N_('Name of the movement (e.g.: "Andante con moto").'),
+        is_from_mb=False,
     ),
     TagVar(
         'movementnumber',
         shortdesc=N_('Movement Number'),
+        longdesc=N_(
+            'Movement number in Arabic numerals (e.g.: "2"). Players explicitly supporting this tag will often '
+            'display it in Roman numerals (e.g.: "II").'
+        ),
+        is_from_mb=False,
     ),
     TagVar(
         'movementtotal',
         shortdesc=N_('Movement Count'),
+        longdesc=N_('Total number of movements in the work (e.g.: "4").'),
+        is_from_mb=False,
     ),
     TagVar(
         'multiartist',
-        shortdesc=N_('FIXME:multiartist'),
+        shortdesc=N_('Multiple Artists'),
+        longdesc=N_(
+            'Set to 1 if not all of the tracks on the album have the same primary artist, otherwise empty.'
+        ),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'musicbrainz_albumartistid',
-        shortdesc=N_('MusicBrainz Release Artist Id'),
+        shortdesc=N_('Release Artist MBID'),
+        longdesc=N_('The MusicBrainz Identifiers (MBIDs) for the release artists.'),
+        is_multi_value=True,
     ),
     TagVar(
         'musicbrainz_albumid',
-        shortdesc=N_('MusicBrainz Release Id'),
+        shortdesc=N_('Release MBID'),
+        longdesc=N_('The MusicBrainz Identifier (MBID) for the release.'),
     ),
     TagVar(
         'musicbrainz_artistid',
-        shortdesc=N_('MusicBrainz Artist Id'),
+        shortdesc=N_('Artist MBID'),
+        longdesc=N_('The MusicBrainz Identifiers (MBIDs) for the track artists.'),
+        is_multi_value=True,
     ),
     TagVar(
         'musicbrainz_discid',
-        shortdesc=N_('MusicBrainz Disc Id'),
+        shortdesc=N_('Disc MBID'),
```

Comment:
There is a little confusion here, that's not an MBID, but a MusicBrainz discid as per https://musicbrainz.org/doc/Disc_ID_Calculation.
This one is a 20 byte SHA-1 signature is converted into a Base64 encoded string of printable ASCII characters, so 28 characters-long string

Also below, we have the discid, as used per FreeDB & CDDB in the past, and still calculated by certain programs:
https://fr.wikipedia.org/wiki/DiscId
This one is 8 hex chars.

![image](https://github.com/user-attachments/assets/0a2a897b-77ce-4915-b9a3-7537e89d6914)

Perhaps we should rename this one "FreeDB discid" and the other "MusicBrainz DiscID".